adcc/AdcMatrix.py

```diff
@@ -165,8 +164,7 @@ def __init__(self, method, hf_or_mp, block_orders=None, intermediates=None,
                                       variant=variant)
                 for block, order in self.block_orders.items() if order is not None
             }
-            # TODO Rename to self.block in 0.16.0
```

Comment:
So we renamed it to `self.blocks` instead of `self.block`? It doesn't really matter. I just want to be sure.

Reply:
I think "blocks" is more natural for this dictionary?